core/trino-main/src/test/java/io/trino/execution/resourcegroups/TestResourceGroups.java

```diff
@@ -1291,6 +1291,48 @@ public void testGetResourceGroupStateInfo()
         assertThat(queryInfo.getResourceGroupId()).isEqualTo(Optional.of(rootB.getId()));
     }
 
+    @Test
+    public void testStartedQueries()
+    {
+        InternalResourceGroup root = new InternalResourceGroup("root", (_, _) -> {}, directExecutor())
+        {
+            @Override
+            public void triggerProcessQueuedQueries()
+            {
+                // No op to allow the test fine-grained control about when to trigger the next query.
+            }
+        };
+        var rootA = root.getOrCreateSubGroup("a");
```

Comment:
In the Trino codebase, explicit types are generally preferred over `var`: https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#avoid-var

Reply:
Oops, thanks for informing me!